openoa/utils/plot.py

```diff
@@ -1932,3 +1932,284 @@ def plot_wake_losses(
+            first dimension should contain results from different Monte Carlo iterations and 95%
+            confidence intervals will be plotted. The last dimension contains the optimal curve
+            fit parameters.
+        mean_vane_angle_ws (:obj:`np.ndarray`): Array containing mean wind vane angles for each
+            wind speed bin in the `ws_bins` argument (degrees).
+        yaw_misalignment_ws (:obj:`np.ndarray`): 1D or 2D array containing yaw misalignment values
+            for each wind speed bin in the `ws_bins` argument (degrees). If a 2D array is provided,
+            the first dimension should contain results from different Monte Carlo iterations and
+            95% confidence intervals will be plotted.
+        turbine_id (str, optional): Name of turbine for which yaw misalignment data are
+            provided. Used to determine title and plot axis labels. Defaults to None.
+        power_performance_label (str, optional): The label to use for the power performance (y) axis.
+            Defaults to "Normalized Cp (-)".
+        xlim (:obj:`tuple[float, float]`, optional): A tuple of floats representing the x-axis
+            wind vane angle plotting display limits (degrees). Defaults to (None, None).
+        ylim (:obj:`tuple[float, float]`, optional): A tuple of the y-axis plotting display limits
+            for the power performance vs. wind vane plots. Defaults to (None, None).
+        return_fig (:obj:`bool`, optional): Flag to return the figure and axes objects. Defaults to False.
+        figure_kwargs (:obj:`dict`, optional): Additional figure instantiation keyword arguments
+            that are passed to `plt.figure()`. Defaults to None.
+        plot_kwargs_curve (:obj:`dict`, optional): Additional plotting keyword arguments that are passed to
+            `ax.plot()` for plotting lines for the power performance vs. wind vane plots. Defaults to {}.
+        plot_kwargs_line (:obj:`dict`, optional): Additional plotting keyword arguments that are passed to
+            `ax.plot()` for plotting vertical lines indicating mean vane angle and vane angle where
+            power is maximized. Defaults to {}.
+        plot_kwargs_fill (:obj:`dict`, optional): If `UQ` is True, additional plotting keyword arguments
+            that are passed to `ax.fill_between()` for plotting shading regions for 95% confidence
+            intervals for power performance vs. wind vane. Defaults to {}.
+        legend_kwargs (:obj:`dict`, optional): Additional legend keyword arguments that are passed to
+            `ax.legend()` for the power performance vs. wind vane plots. Defaults to {}.
+    Returns:
+        None | tuple[matplotlib.pyplot.Figure, matplotlib.pyplot.Axes]:
+            If `return_fig` is True, then the figure and axes object(s) corresponding to the yaw
+            misalignment plots are returned for further tinkering/saving.
+    """
+
+    from openoa.analysis.yaw_misalignment import cos_curve
```

Comment:
I think it might be best to have this method in the `utils` subpackage, but I'm not totally sure where it belongs in there.

Reply:
As we discussed, I left this alone for now.